apps/web/src/pages/cambridge/index.tsx

```diff
@@ -33,7 +33,7 @@ const IndexPage = () => (
         ]}
         right={[
           {
-            text: 'Donate',
+            text: 'Get involved',
             href: 'donate/',
```

Comment:
Do you want to also update this link - e.g. linking to a register as a rep form etc.?